Library/Homebrew/cask/lib/hbc/dsl/appcast.rb

```diff
@@ -9,6 +11,24 @@ def initialize(uri, parameters = {})
         @checkpoint     = @parameters[:checkpoint]
       end
 
+      def calculate_checkpoint
+        result = SystemCommand.run("/usr/bin/curl", args: ["--compressed", "--location", "--user-agent", URL::FAKE_USER_AGENT, @uri], print_stderr: false)
+
+        checkpoint = if result.success?
+          processed_appcast_text = result.stdout.gsub(%r{<pubDate>[^<]*</pubDate>}, "")
+
+          # This step is necessary to replicate running `sed` from the command line
```

Comment:
Just noticed it also needs the `delete("\n")` step before `gsub`.

There’s no harm in messing with current values, since those will be outdated anyway, for missing the `delete` step.